src/tools/rbd_mirror/PoolMetaCache.cc

```diff
@@ -63,20 +63,21 @@ int PoolMetaCache::get_remote_pool_meta(
 }
 
 void PoolMetaCache::set_remote_pool_meta(
-    int64_t pool_id,
+    const std::string& fsid, int64_t pool_id,
     const RemotePoolMeta& remote_pool_meta) {
-  dout(15) << "pool_id=" << pool_id << ", "
+  dout(15) << "fsid=" << fsid << ", pool_id=" << pool_id << ", "
            << "remote_pool_meta=" << remote_pool_meta << dendl;
```

Comment:
Nit: since this statement is being touched anyway
```suggestion
  dout(15) << "fsid=" << fsid << ", pool_id=" << pool_id <<
           << ", remote_pool_meta=" << remote_pool_meta << dendl;
```